cmd/api-linter/cli.go

```diff
@@ -55,10 +55,10 @@ func runCLI(rules lint.Rules, configs lint.Configs, args []string) error {
 					Value: "protoc",
 					Usage: "protocol compiler path",
 				},
-				cli.StringFlag{
+				cli.StringSliceFlag{
 					Name:  "proto_path",
-					Value: ".",
-					Usage: "the directory in which for protoc to search for imports",
+					Value: &cli.StringSlice{"."},
```

Comment:
What's the behavior of this value? If the user specifies value(s) will they override this slice or append to it?

Reply:
Since 2016, it overrides: https://github.com/urfave/cli/issues/160